### ISSUE TITLE
Configure default terminalWidth for non TTY environment

### ISSUE THREAD

**Issue by abourdin** [AriaMinaei/RenderKid#14]
Here in the code, the terminal width is defaulted to 80 for non-TTY terminals:

https://github.com/AriaMinaei/RenderKid/blob/e8c9744ac499a09219413177f6d45a24c4d346b5/src/tools.coffee#L83

This is causing lines to be cut when using in a context where Node is run using terminal redirection (WebStorm run console, AWS CloudWatch logs, ...)

Would it be possible to introduce an environment variable or a way to change this default value? If yes, I can open a pull-request for that.
Environment variable could be something like RENDERKID_DEFAULT_TERMINAL_WIDTH.

**Comment by RayBenefield** [comment]
Would like to echo this need to run on non-TTY 😁

**Comment by AriaMinaei** [comment]
> Would it be possible to introduce an environment variable

@abourdin  That sounds doable but is there a better, automated way to do this? I want to avoid complicating people's setup as much as possible.

**Comment by RayBenefield** [comment]
Yes please. Ran into this again trying to run things with concurrently 😭😭😭
